app/controllers/application_controller/sysprep_answer_file.rb

```diff
@@ -5,7 +5,7 @@ def upload_sysprep_file
       @upload_sysprep_file = true
       @edit = session[:edit]
       build_grid
-      if params.fetch_path(:upload, :file).respond_to?(:read)
+      if params.dig(:upload, :file).respond_to?(:read)
```

Comment:
This one causes the test failure...

     Failure/Error: params.dig(:upload, :file).respond_to?(:read)
     
     TypeError:
       String does not have #dig method

looks like there is a difference between fetch_path and dig:

```
{:foo => ""}.fetch_path(:foo, :bar)
=> nil

{:foo => ""}.dig(:foo, :bar)
TypeError: String does not have #dig method
```

Reply:
Awesome.  I knew it was too good to be true as I remembered they're not exactly the same... thanks, I'll look into it.